src/lib/types.ts

```diff
@@ -158,18 +160,53 @@ export interface v3ProjectsAttributes {
   type: string;
 }
 
-export interface v3ProjectsRelationships {
-  importingUser: v3ProjectsRelashionshipData;
-  org: v3ProjectsRelashionshipData;
-  owner: v3ProjectsRelashionshipData;
-  target: v3ProjectsRelashionshipData;
+export interface RESTProjectsRelationships {
+  importingUser: RESTProjectsRelashionshipData;
+  org: RESTProjectsRelashionshipData;
+  owner: RESTProjectsRelashionshipData;
+  target: RESTProjectsRelashionshipData;
 }
 
-export interface v3ProjectsRelashionshipData {
+export interface RESTProjectsRelashionshipData {
   data: {
     id: string;
     type: string;
   }
   links: unknown;
   meta: unknown;
 }
+
+export interface SnykTargetRESTRelationships {
+  org: {
+    data : {
+      id: string;
+      type: string;
```

Comment:
What is this "type" entry?

Reply:
it is the "org" for an org, project for a "project".